deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -121,8 +121,7 @@
               - /bin/sh
               - -c
               - >
-                printf "GET /healthz HTTP/1.0\r\n\r\n" |
-                nc -q 2 -U /var/vcap/data/cloud_controller_ng/cloud_controller.sock |
+                curl -sI --unix-socket /var/vcap/data/cloud_controller_ng/cloud_controller.sock http:/healthz |
```

Comment:
:tada: we finally have `curl --unix-socket`

Doesn't that mean we can just use `curl --fail` and not need the pipe to `grep`? And then we can drop the `/bin/sh` too… at which point long args seem better.